TKGame/TKGame.cs

```diff
@@ -46,11 +46,11 @@ public class TKGame : Game
         
         // TODO: Refactor out of the main TKGame class
         private static string currentStageName = "defaultStage" + ".json";
-        Stage currentStage;
+        internal Stage currentStage;
         //Stage leftStage;
         //Stage rightStage;
         int screenWidth, screenHeight;
-        bool paused = false;
+        public static bool paused = true;
```

Comment:
Yeah definitely a good call, I've been thinking about this since I've had to spawn the player higher than the ground b/c physics before the game was visible.